### DISCUSSION TITLE
Option to fade out the read articles in addition to hiding them

### DISCUSSION THREAD

**Discussion by selvarajsubramanian** [dailydotdev/daily#411] [Feature Requests]
Hi Team, I feel it would be nice if you could provide an option to fade out the already-read articles/cards in addition to hiding them.
Any thoughts?

Thanks,
Selva

**Comment by idoshamun** [comment]
Hi @selvarajsubramanian,

Thanks for the suggestion!
If you are interested in seeing articles you read, you can visit your reading history:
https://app.daily.dev/history
Let me know if it solves the issue :)

**Reply by selvarajsubramanian** [reply]
Thanks idoshamun for the reply and suggestion, I just want to declutter my view and easily differentiate between the read and unread articles while scrolling down the feed. I can see its currently supported by differentiating the border around the card but its very light.

**Reply by idoshamun** [reply]
I'll forward this feedback to the team :)
Anyway, after a few minutes since you last read the article it should be removed from your feed. Is it not?

**Reply by selvarajsubramanian** [reply]
oh okay, I just noticed that already-read articles are getting removed from the feed automatically. Thanks again idoshamun :)
